department-of-veterans-affairs/caseflow
# Do not save two claimants during intake, use last claimant if there are two

### Description
If someone goes back and edits the claimant during intake (on the review request page), a second claimant is added to the decision review.  The second claimant is the correct claimant, before we were referencing the first claimant.

This PR:
- Fixes the bug where multiple claimants are created for a decision review
- Changes the code to use the last claimant instead of the first claimant. This is to be safe for the small number of decision reviews that were already created with two claimants.

### Testing Plan
1. Start a decision review
2. Add a claimant, proceed to add issues page
3. Go back, change the claimant, proceed to add issues page
4. Check that the second claimant is the reviews claimant, and that there are no other claimants

Also:
If there are multiple claimants, check that the last claimant is used for establishing a claim and creating a decision document.
